datafusion/functions-nested/src/reverse.rs

```diff
@@ -183,6 +195,75 @@ fn general_array_reverse<O: OffsetSizeTrait + TryFrom<i64>>(
     )?))
 }
 
+fn list_view_reverse<O: OffsetSizeTrait + TryFrom<i64>>(
+    array: &GenericListViewArray<O>,
+    field: &FieldRef,
+) -> Result<ArrayRef> {
+    let (_, offsets, sizes, values, nulls) = array.clone().into_parts();
```

Comment:
```suggestion
    let offsets = array.offsets();
    let values = array.values();
    let sizes = array.sizes();
```

I don't think there's a strict need to clone and deconstruct the array, can just grab constituent parts like so

Reply:
Good point, thanks. I do end up using all except `field`, but this  is more readable